### DISCUSSION TITLE
Do usgs_satapi_aws credentials have changed?

### DISCUSSION THREAD

**Discussion by remi-braun** [CS-SI/eodag#499] [Q&A] [answered]
Hello,

When I try to download some LANDSAT_C2L1data from `usgs_satapi_aws`, it is said that I have an authentication issue:
![2022-09-01_15h38_21](https://user-images.githubusercontent.com/67311115/187927974-bedb3780-7713-4b7e-9f86-6d005a294134.png)

Do you know if they changed something on their side ? 
Or have I broken my chain somehow ?

(I didn't want to set an issue till I am sure it isn't my fault 😅 )

PS: My AWS account is set and works like a charm as I manage to download on `earth_search` without any problem
PPS: the `usgs_satapi_aws` link (https://landsatlook.usgs.gov/sat-api/) seems broken

**Comment by sbrunato** [comment, marked as answer]
Hello @remi-braun , do you still have this error ? The bucket `usgs-landsat` is valid, so the message raised (`NoSuchBucket`) may mean that the service was temporarily not available on AWS

**Reply by remi-braun** [reply]
Yes this works now, I only have a WARNING left:

```
2022-09-12 10:05:12 - [WARNING] - Unexpected error: An error occurred (NoSuchBucket) when calling the ListObjects operation: The specified bucket does not exist
2022-09-12 10:05:12 - [WARNING] - Skipping stac-browser//collection02/level-1/standard/oli-tirs/2022/152/041/LC09_L1TP_152041_20220828_20220828_02_T1/LC09_L1TP_152041_20220828_20220828_02_T1
```

This may not be important, but I wanted to know if it happens to you too 😅
